src/env.cc

```diff
@@ -24,6 +25,11 @@ using v8::StackTrace;
 using v8::String;
 using v8::Value;
 
+const int kNodeContextTag = 0x6e6f64;
+void* kNodeContextTagPtr = const_cast<void*>(
+  static_cast<const void*>(&kNodeContextTag)
```

Comment:
Ditto.

Reply:
done (the parenthesis part, the indent used looks like 2 spaces to me? or is 4 a special rule for these cases?)

Reply:
If it’s a block/conditional body/function body, 2 spaces, otherwise 4 for statement continuations.

We try to document this at https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md, if you have any suggestions for improving the clarity of that document please say so :)

Reply:
ah thanks, missed that one. fixed!